Lesson16/baljan/script.js

```diff
@@ -0,0 +1,412 @@
+/*
+8. Print Each Character of a String (For-of)
+   - Define a function `printCharacters(str)` that uses a for-of loop
+     to log each character in the string on a separate line.
+*/
+
+function printCharacters(str) {
+  for (char of str) {
```

Comment:
You need to add the keyword 'const' before the variable name 'char' inside the (). Otherwise, JS would try to instantiate a variable itself which sometimes can cause unexpected behavior.

```suggestion
  for (const char of str) {
```